x-pack/plugins/security/server/config.ts

```diff
@@ -39,17 +39,31 @@ export const ConfigSchema = schema.object(
       lifespan: schema.nullable(schema.duration()),
     }),
     secureCookies: schema.boolean({ defaultValue: false }),
-    authc: schema.object({
-      providers: schema.arrayOf(schema.string(), { defaultValue: ['basic'], minSize: 1 }),
-      oidc: providerOptionsSchema('oidc', schema.object({ realm: schema.string() })),
-      saml: providerOptionsSchema(
-        'saml',
-        schema.object({
-          realm: schema.string(),
-          maxRedirectURLSize: schema.byteSize({ defaultValue: '2kb' }),
-        })
-      ),
-    }),
+    authc: schema.object(
+      {
+        providers: schema.arrayOf(schema.string(), { defaultValue: ['basic'], minSize: 1 }),
+        oidc: providerOptionsSchema('oidc', schema.object({ realm: schema.string() })),
+        saml: providerOptionsSchema(
+          'saml',
+          schema.object({
+            realm: schema.string(),
+            maxRedirectURLSize: schema.byteSize({ defaultValue: '2kb' }),
+          })
+        ),
+        http: schema.object({
+          enabled: schema.boolean({ defaultValue: true }),
+          autoSchemesEnabled: schema.boolean({ defaultValue: true }),
```

Comment:
__note:__ didn't manage to find a better and more obvious name for this setting....

Reply:
Good to know, thanks. Will keep it as it's now then.